packages/prisma/extensions/disallow-undefined-in-delete-update-many.ts

```diff
@@ -0,0 +1,31 @@
+import { Prisma } from "@prisma/client";
+
+const checkUndefinedInValue = (where: any) => {
+  if (where) {
+    for (const key in where) {
+      // INFO: Since this is for $allModels, we don't have a way to get the correct
+      // where type
+      const whereInput = where[key as any] as any;
+      if (whereInput.hasOwnProperty("in") && typeof whereInput.in === "undefined") {
```

Comment:
There is a possibility for whereInput to be `undefined` I think like someone could accidentally pass `where:{id: undefined}` and then it would crash. Not sure what should we do in that case though.